### PR TITLE
Linux chapter navigating

### PR DESCRIPTION
**Pull request by redfast00**

### CODE REVIEW

**Comment by boegel** [comment]
@redfast00 These changes are good to go, but I think the Travis config doesn't run the version check yet on the Linux introduction, we should change that.

Can you look into that first (please make it a separate PR), see #184

**Comment by redfast00** [comment]
@boegel #184 was merged,  do I now do the final commit (run version script)?

**Comment by boegel** [comment]
@redfast00 Yes, sounds good to me.

File: intro-Linux/ch_navigating.tex [hunk 1]
```diff
@@ -115,7 +116,7 @@ \section{Absolute vs relative file paths}
 with ``/''), which is also called the \emph{root} of the filesystem.
 
 Example: absolute path to your home directory:
-``%\homedir%''.
+``\homedir''.
```
Comment (boegel) [inline intro-Linux/ch_navigating.tex]: @redfast00 Ah, we should probably have used `\texttt{\homedir}` here?

Please fix that in some other PR where it makes sense (one that deal with typos/formatting in another chapter)

**Pull request merged**